web-api/src/persistence/postgres/messages/getSectionOutboxMessages.ts

```diff
@@ -0,0 +1,28 @@
+import { MessageResult } from '@shared/business/entities/MessageResult';
+import { calculateISODate } from '@shared/business/utilities/DateHandler';
+import { getDbReader } from '@web-api/database';
+import { transformNullToUndefined } from '@web-api/persistence/postgres/utils/transformNullToUndefined';
+
+export const getSectionOutboxMessages = async ({
+  section,
+}: {
+  applicationContext: IApplicationContext;
+  section: string;
+}): Promise<MessageResult[]> => {
+  const filterDate = calculateISODate({ howMuch: -7 });
+
+  const messages = await getDbReader(reader =>
+    reader
+      .selectFrom('message as m')
+      .leftJoin('case as c', 'c.docketNumber', 'm.docketNumber')
+      .where('m.fromSection', '=', section)
+      .where('m.createdAt', '>=', filterDate)
+      .selectAll()
+      .select('m.docketNumber')
```

Comment:
I might be misunderstanding the kysely API, but is a selectAll necessary if we are following up with a more specific select?

Reply:
(Here and elsewhere in the persistence methods)

Reply:
Also, I don't know how the query optimizer works, but filtering before the join would probably be more efficient. (But it's completely possible the query optimizer does this already.)